recipes/omniscidb-cuda/recipe/meta.yaml

```diff
@@ -0,0 +1,97 @@
+{% set name = "omniscidb-cuda" %}
+{% set version = "4.8.0" %}
+{% set sha256 = "06ec0990d362963fc5e02ad64bc90d99ad0d226014354de4566af267d33443b1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/omnisci/omniscidb/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    Skip-installing-Boost-libs.patch
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    #- {{ compiler('c') }}
+    - boost-cpp 1.68.*        # to avoid cmake picking up boost-cpp 1.70+
+    - cmake
+    - make
+    - go
+    - maven
+  host:
+    - {{ compiler('cxx') }}
+    #- {{ compiler('c') }}
+    - boost-cpp 1.68.*
+    - llvmdev
+    - clangdev
+    - double-conversion
+    - snappy
+    - gflags
+    - glog
+    - libarchive
+    - libkml
+    - libpng
+    - blosc
+    - libgdal >=2.3
+    - arrow-cpp >=0.12.1
+    - thrift-cpp >=0.11.0
+    - ncurses
+    - flex
+    - bisonpp
+    - openssl
+    - librdkafka
+    - cudatoolkit-dev
+    - openldap
+    # Required by cudatoolkit-dev:
+    - wget
+  run:
+    - boost-cpp 1.68.*
+    - arrow-cpp >=0.12.1
+    - snappy
+    - double-conversion
+    - libgdal >=2.3
+    - glog
+    - gflags
+    - openjdk 8.*
```

Comment:
No need for an asterix, conda does exactly that automatically.

Reply:
Without the asterix, there is a message:
```
WARNING:conda_build.utils:Adding .* to spec 'openjdk 8' to ensure satisfiability.  Please consider putting {{ var_name }}.* or some relational operator (>/</>=/<=) on this spec in meta.yaml, or if req is also a build req, using {{ pin_compatible() }} jinja2 function instead.  See https://conda.io/docs/user-guide/tasks/build-packages/variants.html#pinning-at-the-variant-level
```
Should the message be ignored?

Reply:
No, I just didn't know that the convention is now the other way around